managed-services/iot-hub/reference-content/certificate.mdx

```diff
@@ -0,0 +1,67 @@
+---
+meta:
+  title: Updating your Endpoint's Certificate Authority 
+  description: This page provides important information about the Scaleway IoT Hub Certificate Authority.
+content:
+  h1: Certificate Authority update
+  paragraph: This page provides important information about the Scaleway IoT Hub Certificate Authority.
+dates:
+  validation: 2023-04-27
+  posted: 2023-04-27
+categories:
+  - managed-services
+tags: iot iot-hub ca certificate authority update
+---
+
+As a security measure, Scaleway IoT Hub was designed with a mutual Transport Layer Security (TLS) as the default authentication protocol. This means each device needs both a certificate and key pair to connect to IoT Hub.
+
+Using a trusted Certificate Authority (CA) to issue digital certificates is essential to guarantee a secure Transport Layer Security (TLS) connection between a device and the IoT Hub. A digital certificate contains important information about the server's identify that can help verify its authenticity, then allowing the device to use it to establish a secure connection with the server.
+
+However, the current Certificate Authority of the IoT Hub server endpoint is set to expire **in February 2024**.
+
+The information provided on this page explains what the migration entails and how to successfully transition to the new CA.
+
+<Message type="note">
+  It is possible to replace the Scaleway-generated default certificate with either a self-generated or third-party certificate. If your devices do not use TLS, the information contained within this page will not pertain to you.
```

Comment:
The first sentence is not useful (and can be misleading) as we are not talking about the same certificate at all.
It's probably simpler to just keep the last part in this note:
> If your devices do not use TLS, the information contained within this page will not pertain to you.